sections/main-product.liquid

```diff
@@ -1,729 +1,96 @@
-<section
-  id="MainProduct-{{ section.id }}"
-  class="page-width section-{{ section.id }}-padding"
-  data-section="{{ section.id }}"
->
-  {{ 'section-main-product.css' | asset_url | stylesheet_tag }}
-  {{ 'component-accordion.css' | asset_url | stylesheet_tag }}
-  {{ 'component-price.css' | asset_url | stylesheet_tag }}
-  {{ 'component-rte.css' | asset_url | stylesheet_tag }}
-  {{ 'component-slider.css' | asset_url | stylesheet_tag }}
-  {{ 'component-rating.css' | asset_url | stylesheet_tag }}
-  {{ 'component-loading-overlay.css' | asset_url | stylesheet_tag }}
-  {{ 'component-deferred-media.css' | asset_url | stylesheet_tag }}
+<div class="color-{{ section.settings.color_scheme }}">
```

Comment:
Im not sure how I feel about having the div wrapping everything 🤔 But due to how we apply the padding on the `<section` element I guess that makes the most sense. Otherwise it could be like this: 
![](https://screenshot.click/12-17-v4piu-4enr8.png)


I actually wonder if they should be both `div` element instead 🤔 Cause at the moment we actually have a `section` element within another `section` element. 

![](https://screenshot.click/12-19-lth2s-55r3f.png)